generators/CoreCLRStatefulActor/index.js

```diff
@@ -95,16 +95,36 @@ var ClassGenerator = generators.Base.extend({
     var codePath = path.join(appPackage , appPackagePath, servicePackage, 'Code');
     var testCodePath = path.join(appPackage , serviceProjName + 'TestClient' );
 
+    var is_Windows = (process.platform=='win32');
+    var is_Linux = (process.platform=='linux');
+    var is_mac = (process.platform=='darwin');
+
+    var extension1;
+    var extension2;
+    var serviceManifestFile;
+    if(is_Windows)
+    {
+      extension1 = '.ps1';
+      extension2 = '.cmd';
+      serviceManifestFile = 'ServiceManifest.xml';
+    }
+    else if(is_Linux){
```

Comment:
What about MAC?

Reply:
Added support for MAC and it has servicemanifest same as that of windows (avoiding entrypoint)